src/components/ZoomedImage.tsx

```diff
@@ -7,32 +9,76 @@ interface ZoomedImageProps {
   width: number;
   height: number;
   close: () => void;
+  previous: () => void;
+  next: () => void;
+  zoomedImageIndex: number;
+  images?: Asset[];
 }
 
-const ZoomedImage = ({ src, alt, width, height, close }: ZoomedImageProps) => {
+const ZoomedImage = ({
+  src,
+  alt,
+  width,
+  height,
+  close,
+  previous,
+  next,
+  zoomedImageIndex,
+  images = [],
```

Comment:
passing all images does not seem like very useful. Instead Pass only the props this particular component needs. here it could be imagesLength: number